aspnet/tutorials/nano-server/enable-ancm.ps1

```diff
@@ -11,14 +11,14 @@ $section = $appHostconfig.GetSection("system.webServer/handlers")
 $section.OverrideMode="Allow"
 
 # Add httpPlatform section to system.webServer
-$sectionHttpPlatform = $appHostConfig.RootSectionGroup.SectionGroups["system.webServer"].Sections.Add("httpPlatform")
```

Comment:
Looks like you already have the PS script. Could you add a line to backup existing config before making the changes.

```
copy C:\Windows\System32\inetsrv\config\applicationHost.config C:\Windows\System32\inetsrv\config\applicationHost_BeforeInstallingANCM.config

.....changes to install ancm....

copy C:\Windows\System32\inetsrv\config\applicationHost.config C:\Windows\System32\inetsrv\config\applicationHost_AfterInstallingANCM.config


```